pkg/actions/irsa/irsa.go

```diff
@@ -49,3 +48,10 @@ func doTasks(taskTree *tasks.TaskTree) error {
 	}
 	return nil
 }
+
+// logPlanModeWarning will log a message to inform user that they are in plan-mode
+func logPlanModeWarning(plan bool) {
+	if plan {
+		logger.Warning("no changes were applied, run again with '--approve' to apply the changes")
+	}
+}
```

Comment:
A new shared library would be good, but no doubt that is how `ctl/utils` started 😆 . I had to specifically move it out here to prevent some dodgy import cycles from occurring 🙄

Reply:
> but no doubt that is how ctl/utils started

Totally! not sure which is worse move: shared lib which starts cute but grows to be ugly, or repeated code

> I had to specifically move it out here to prevent some dodgy import cycles from occurring
Ah makes sense.